rest/changes_api.go

```diff
@@ -248,6 +248,15 @@ func (h *handler) handleChanges() error {
 		feed = "normal"
 	}
 
+	if feed != "normal" {
+		release, err := h.server.limitConcurrentReplications(h.rqCtx)
```

Comment:
I'm on the fence about whether we omit one-shot ("normal") feeds - a one-shot changes feed can be relatively long-running.